.claude/commands/tests/test_exportcommands.py

```
@@ -781,6 +781,100 @@ def test_export_directory_uses_rsync_when_available(self):
+        cls.project_root = os.path.abspath(
+            os.path.join(os.path.dirname(__file__), '..', '..', '..')
+        )
+        cls.scripts_root = os.path.join(cls.project_root, 'scripts')
+        cls.claude_scripts = os.path.join(cls.project_root, '.claude', 'scripts')
+
+    def test_auth_cli_mjs_present_in_claude_scripts(self):
+        """auth-cli.mjs must exist in .claude/scripts/ (source for export)."""
+        path = os.path.join(self.claude_scripts, 'auth-cli.mjs')
+        self.assertTrue(
+            os.path.isfile(path),
+            f"Missing .claude/scripts/auth-cli.mjs — required by secondo-cli.sh and /secondo"
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)